src/Resources/views/CRUD/Association/edit_one_to_many_inline_table.html.twig

```diff
@@ -23,9 +23,12 @@ file that was distributed with this source code.
                             style="display:none;"
                         {% endif %}
                     >
-                        {{ nested_field.vars.label|trans({}, nested_field.vars['sonata_admin'].admin.translationDomain
-                            |default(nested_field.vars.translation_domain)
-                        ) }}
+                        {% if nested_field.vars.translation_domain is same as(false) %}
+                            {{ nested_field.vars.label }}
+                        {% else %}
+                            {% set translationDomain = nested_field.vars.translation_domain|default(nested_field.vars['sonata_admin'].admin.translationDomain) %}
```

Comment:
At first sight, there is no other variable using camel case in this file, excepting this property:
```twig
sonata_admin.field_description.associationadmin.formfielddescriptions[field_name].translationDomain
```